client/blocks/upload-image/constants.js

```diff
@@ -0,0 +1,7 @@
+export const ALLOWED_FILE_EXTENSIONS = [
```

Comment:
Hmm, how does this compare with say `media/constants.js`?

It appears that we have more image file types we support
```jsx
	// Image formats
	jpg: 'image/jpeg',
	jpeg: 'image/jpeg',
	jpe: 'image/jpeg',
	gif: 'image/gif',
	png: 'image/png',
	bmp: 'image/bmp',
	tif: 'image/tiff',
	tiff: 'image/tiff',
	ico: 'image/x-icon',
```